nengo/spa/input.py

```diff
@@ -40,7 +40,10 @@ def on_add(self, spa):
         Module.on_add(self, spa)
 
         for name, value in iteritems(self.kwargs):
-            target, vocab = spa.get_module_input(name)
+            try:
+                target, vocab = spa.get_module_input(name)
+            except TypeError:
+                raise ValueError('Could not find input called "%s"' % name)
```

Comment:
We do a lot of this in SPA: catching errors and turning them into other ones. It bit me in the butt yesterday, when some code I was writing had a NameError in it, which then got caught and turned into something different. I think we should avoid it when possible. Here, can't we just have `spa.get_module_input` return the correct ValueError if it can't find `name`? That seems much cleaner to me.